src/Components/Label/index.css

```diff
@@ -0,0 +1,14 @@
+.LabelTitle {
```

Comment:
Why `LabelTtitle` and not just `Label`?

Reply:
You are right. We actually agreed on Label only.